test-suite/src/test/require-jdk7/org/scalajs/testsuite/javalib/util/CollectionsTestOnJDK7.java

```diff
@@ -5,12 +5,14 @@
 ** /____/\___/_/ |_/____/_/ | |__/ /____/                               **
 **                          |/____/                                     **
 \*                                                                      */
-package org.scalajs.testsuite.javalib
+package org.scalajs.testsuite.javalib.util
 
 import java.{util => ju}
```

Comment:
I suggest to use this opportunity to fix the spaces into normal spaces ;) Boy Scout rule.